FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/SendPasswordProtectedMessageFlowTest.kt

```diff
@@ -230,14 +252,38 @@ class SendPasswordProtectedMessageFlowTest : BaseDraftsGmailAPIFlowTest() {
         ByteArrayInputStream(decryptedContent?.toByteArray())
       )
 
+      //we should be sure that me have right recipients in MIME message. BCC should be empty here.
+      assertEquals(
+        addAccountToDatabaseRule.account.email,
+        (mimeMessage.from.first() as InternetAddress).address
+      )
+      assertArrayEquals(
+        arrayOf(TO_RECIPIENT_WITHOUT_PUBLIC_KEY),
+        mimeMessage.getRecipients(Message.RecipientType.TO)
+          .map { (it as InternetAddress).address }
+          .toTypedArray()
+      )
+      assertArrayEquals(
+        emptyArray(),
+        mimeMessage.getRecipients(Message.RecipientType.CC)
+          ?.map { (it as InternetAddress).address }
+          ?.toTypedArray() ?: emptyArray()
+      )
+      assertArrayEquals(
+        emptyArray(),
+        mimeMessage.getRecipients(Message.RecipientType.BCC)
+          ?.map { (it as InternetAddress).address }
+          ?.toTypedArray() ?: emptyArray()
+      )
+
```

Comment:
Better to make common function like this and use it like below.

```kt
assertArrayEquals(arrayOf(TO_RECIPIENT_WITHOUT_PUBLIC_KEY), getEmailAddresses(Message.RecipientType.TO))
```


```kt
fun getEmailAddresses(type: Message.RecipientType): Array<String> {
    return mimeMessage.getRecipients(type)
        ?.map { (it as InternetAddress).address }
        ?.toTypedArray() ?: emptyArray()
}

```

Reply:
Additionally, I think we better need to verify that the `CC` recipient is not hidden like the `BCC` recipient.